server/routes/index.ts

```diff
@@ -67,7 +67,6 @@ export function defineRoutes(router: IRouter, esClient: IClusterClient) {
   });
 
   const roleSchema = schema.object({
-    description: schema.string(),
```

Comment:
Is this intentional?

Reply:
Yes since description is not required. It can be removed since it has already be fixed: https://github.com/opendistro-for-elasticsearch/security-kibana-plugin/pull/221/files#diff-23d3a291c8cd7e0967a1ccf38704c83dL70